src/main/scala/com/codacy/client/bitbucket/v2/service/PullRequestServices.scala

```diff
@@ -15,10 +17,12 @@ class PullRequestServices(client: BitbucketClient) {
   def getPullRequests(
       owner: String,
       repository: String,
-      states: Seq[String] = Seq("OPEN")
+      states: Seq[String] = Seq("OPEN"),
+      size: Int = 50
   ): RequestResponse[Seq[PullRequest]] = {
-    val url =
-      s"https://bitbucket.org/api/2.0/repositories/$owner/$repository/pullrequests?pagelen=50&state=${states.mkString("&state=")}"
+    val pullRequestsUrl = generatePullRequestsUrl(owner, repository)
+    val encodedStates = states.map(state => URLEncoder.encode(state, "UTF-8"))
```

Comment:
Codacy found an issue: [The string literal 'UTF-8' appears 3 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)